invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js

```diff
@@ -36,6 +37,11 @@ const RecordVersionItem = ({ item, activeVersion }) => {
             {item.publication_date}
           </p>
         </Grid.Column>
+        <Grid.Column width={12}>
+          <p className="text-muted font-small">
+            {_.get(item.pids, "doi.identifier", "")}
```

Comment:
I don't see the import of `get`: you should always import specific function whenever possible to take advantage of the [tree-shaking feature](https://developer.mozilla.org/en-US/docs/Glossary/Tree_shaking).
```
import _get from 'lodash/get';
```